fritzconnection/lib/fritzwlan.py

```diff
@@ -121,7 +121,7 @@ def _get_wifi_qr_code(instance, kind='svg',
         ssid=instance.ssid,
         password=instance.get_password(),
         security=security,
-        hidden=hidden
+        hidden=instance.is_hidden
```

Comment:
so we can remove the `hidden` option from the signature of `_get_wifi_qr_code`

Reply:
Yes, complexity reduced! By the way, I don't know how you prefer your docstrings, should they tell all the story in retrospect or reflect the situation as it is now? So I can simply remove the hidden explanation from `.. versionadded:: 1.9.0` section or put a new section with the note `hidden` parameter was dropped?